_raml/apps/products/examples/patch_product_productid_request.json

```diff
@@ -7,4 +7,9 @@
   "op": "add",
   "path": "/shortDescription",
   "value": "new arrival"
+},
+{
+  "op": "add",
+  "path": "/deliveryWeight",
+  "value": {"unit":"kg", "amount": 0.45}
```

Comment:
I've seen that currently, the response on the documented patch request ( https://developer.epages.com/apps/api-reference/patch-shopid-products-productid.html ) is:
`"deliveryWeight": {
    "amount": 260,
    "unit": "g"
  }`
So it would be perfect if instead of my previously suggested value, we could change it to match:
`"value": {"unit":"g", "amount": 260}`
(sorry, my fault)